pkgs/by-name/li/liba2i/package.nix

```
@@ -0,0 +1,41 @@
+{ stdenv
+, fetchgit
+, lib
+, pkgconf
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "liba2i";
+  version = "unstable-2024-01-13";
+
+  src = fetchgit {
+    url = "git://www.alejandro-colomar.es/src/alx/alx/liba2i.git";
+    rev = "1d54d788038e671a5f85d79dd932b322247f03e2";
+    hash = "sha256-HqcQq8I91IYzBsFXoMdiivfc5iBCQcM/QmSw/jbuNhM=";
+  };
+
+  postPatch = ''
+    substituteInPlace GNUmakefile \
+      --replace 'SHELL := /usr/bin/env' "" \
+      --replace '.SHELLFLAGS := -S bash -Eeuo pipefail -c' ""
+  '';
+
+  makeFlags = [
+    "DESTDIR=$(out)"
```

```suggestion
    "DESTDIR=${placeholder "out"}"
```

not sure which is preferred tbh but this substitutes it on the Nix side instead of part of Makefile expansion

---

Thanks, make sense.